cmd/nerdctl/main.go

```
@@ -418,6 +423,39 @@ func AddPersistentStringFlag(cmd *cobra.Command, name string, aliases, nonPersis
 	}
 }
 
+// AddPersistentBoolFlag is similar to AddBoolFlag but persistent.
+// See https://github.com/spf13/cobra/blob/master/user_guide.md#persistent-flags to learn what is "persistent".
+func AddPersistentBoolFlag(cmd *cobra.Command, name string, aliases, nonPersistentAliases []string, value bool, env, usage string) {
+	if env != "" {
+		usage = fmt.Sprintf("%s [$%s]", usage, env)
+	}
+	if envV, ok := os.LookupEnv(env); ok {
+		value, _ = strconv.ParseBool(envV)
```

Don't ignore the error

---

how about adding a warning log

---

SGTM